### PR TITLE
10.1.9 release notes

### PR DESCRIPTION
**Pull request by raboof**

### CODE REVIEW

File: docs/src/main/paradox/release-notes/10.1.x.md [hunk 1]
```diff
@@ -2,11 +2,37 @@
 
 ## 10.1.9
 
+10.1.9 is the tenth release in the 10.1.x series of Akka HTTP. See the [announcement](https://akka.io/blog/news/2019/07/17/akka-http-10.1.9-released.html) and the full list of changes below.
+
 ### Changes since 10.1.8
 
+For a full overview you can also see the [10.1.9 milestone](https://github.com/akka/akka-http/milestone/10.1.9?closed=1):
+
+##### akka-http-core
```
Comment (helena) [inline docs/src/main/paradox/release-notes/10.1.x.md]:
```suggestion
#### akka-http-core
```

File: docs/src/main/paradox/release-notes/10.1.x.md [hunk 2]
```diff
@@ -2,11 +2,37 @@
...
+
+* Don't use a https connection pool for http connections (javadsl) [#2562](https://github.com/akka/akka-http/pull/2562)
+
+##### akka-http
```
Comment (helena) [inline docs/src/main/paradox/release-notes/10.1.x.md]:
```suggestion
#### akka-http
```

File: docs/src/main/paradox/release-notes/10.1.x.md (hunk 1 above)
Comment (helena) [inline docs/src/main/paradox/release-notes/10.1.x.md]: Should this and the relevant below be `####` vs `#####`?

Comment (raboof) [reply docs/src/main/paradox/release-notes/10.1.x.md]: I think we sometimes split things up between 'improvements' and 'bugfixes', and always use `#####` whether or not we have that split for that release.

**Pull request merged**